src/wp-includes/class-wp-theme-json.php

```diff
@@ -1717,7 +1717,12 @@ protected static function compute_style_properties( $styles, $settings = array()
 	 * @return string|array Style property value.
 	 */
 	protected static function get_property_value( $styles, $path, $theme_json = null ) {
-		$value = _wp_array_get( $styles, $path );
+		$value = _wp_array_get( $styles, $path, '' );
+
+		// It doesn't make sense to process the value further.
+		if ( in_array( $value, array( '', null ), true ) ) {
```

Comment:
Looks like this was changed after I reviewed.

Please change it to a direct comparison for better performance, `in_array()` is notoriously slow.

```suggestion
		if ( null === $value || '' === $value ) {
```

Reply:
Yes, I changed it because I noticed 2 additional tests that had to be fixed.

Reply:
Fixed in 0d66f734d39e7013a5d31a5916fd1847f88bcc04